.cargo/config.toml

```diff
@@ -1,3 +1,5 @@
 [build]
 target = 'wasm32-wasi'
+
+[target.wasm32-wasi]
```

Comment:
Ah I forgot about this! This can lead to the removal of the `env: RUSTFLAGS:` in the `.github/workflows/main.yml` if you'd like.